Source/UI/NewUI/PaymentForm/Factory/Theme/Generic/Theme+Button.swift

```diff
@@ -0,0 +1,89 @@
+//
+//  Theme+Button.swift
+//  
+//
+//  Created by Alex Ioja-Yang on 09/08/2022.
+//
+
+import UIKit
+
+public extension Theme {
+    
+    /// Theme generated navigation button Style
+    struct NavigationButtonStyle: ElementButtonStyle {
```

Comment:
```suggestion
    public struct NavigationButtonStyle: ElementButtonStyle {
```

Reply:
As the objects are declared within a public extension, everything inherits that access control level.